### ISSUE TITLE
Show modularity of elliptic curves where possible

### ISSUE THREAD

**Issue by JohnCremona** [LMFDB/lmfdb#2003]
Thanks to work of Siksek et al, we know that all elliptic curves over real quadratic fields are modular, and over arbitrary totally real fields there is a simple criterion to check (which conjecturally always holds, of course).  Siksek has written a Magma function to check this, and I am running it on all the curves we have over t.r. fields of degrees 3, 4, 5, 6.  To record that this has been done I'll add a field 'modular' with value True (presumably always), and show this on each curve's home page.

**Comment by JohnCremona** [comment]
All our curves over totally real fields are now proved to be modular.

**Comment by JohnCremona** [comment]
By now SIksek et al have also proved that all curves over totally real cubic fields are modular, so the curve-by-curve check is only needed for degrees 4, 5, 6.

**Comment by JohnCremona** [comment]
I suggest that the web page for these curves just says, in a section headed *Modularity*, "This elliptic curve is modular."  The word modular would be a knowl, perhaps ec.modular,  which would (1) give the definition of what modular means,  (2)(a) give the state-of-the-art concerning which fields over which modularity is known for all curves, (2)(b) for other fields say whether it is possible in principle to prove modularity for individual curves.  That should cover Q, totally real and imaginary quadratic fields, but not (as far as I know) the mixed cubic 3.1.23.1.

Comments about this knowl content?  Do we need more than one knowl?   @jvoight @kedlaya ?

**Comment by jvoight** [comment]
Yes, I think we should display modularity as a property of the elliptic curve, like the others (e.g. base change).  We would likely reproduce this information on the "Langlands picture" page as well.  Along the lines you suggest, it should be text + knowl, and I'm happy to review the knowls as they are written.

**Comment by alozanoroble** [comment]
@JohnCremona was the knowl for "modular" ever created? I can't find it so I don't think so but just checking. If not, I can work on creating the knowl in preparation to add "modularity" as a property to be listed on elliptic curve pages.
